### DISCUSSION TITLE
Can't Use a Model After AutoTrained It

### DISCUSSION THREAD

**Discussion by fatihayUtopia** [huggingface/autotrain-advanced#420] [Q&A] [answered]
hello ı am newbie for aı and hugging face, my problem is:
ı have autotrained a model(fatihay/falconTest) and ı can not use it in any form. I want to use it in inference API. I think the model has lack of some required files as default after autotrain. I think ı should add a config.json but ı don't know how the content of it must be

**Comment by abhishekkrthakur** [comment, marked as answer]
see pinned issue: https://github.com/huggingface/autotrain-advanced/issues/299

**Reply by fatihayUtopia** [reply]
thank you. Your this comment was efficient in that issue "you can also use this space to merge adapters: https://huggingface.co/spaces/autotrain-projects/llm-merge-adapter"

**Reply by abhishekkrthakur** [reply]
from today, you dont need to merge adapters to deploy it on inferencce Endpoints

**Reply by fatihayUtopia** [reply]
but ı still need to merge adapters. How can ı add inference api to model card otherwise? And ı have noticed when ı merge adapters, hf add the BASE model's inference to model card.

(fatihay/gpt2Auto)

And ı can not use my from "gpt2" autotrained model properly. my dataset is attached. ı always get the same answer(outside the dataset) from my autotrained model even if ı was input the same words with my one of the data from the dataset.
Can you help me also with that?
And how can ı learn more and ask a questions? Is there any discord groups or something
Thank you

[dataset.csv](https://github.com/huggingface/autotrain-advanced/files/13798129/dataset.csv)
